assets/pages/container/[id].vue

```diff
@@ -16,11 +16,8 @@ const { id } = defineProps<{ id: string }>();
 const currentContainer = store.currentContainer($$(id));
 const { activeContainers, ready } = storeToRefs(store);
 
-setTitle("loading");
```

Comment:
But when loading this is removed now. Perhaps in `watchEffect` it should check for `ready` and set loading depending on that?

Reply:
I think this conversation could be closed in favor of https://github.com/amir20/dozzle/pull/2365/files#r1304933814